zproject/settings.py

```diff
@@ -214,7 +214,7 @@ def get_secret(key):
                     'USE_WEBSOCKETS': True,
                     'ANALYTICS_LOCK_DIR': "/home/zulip/deployments/analytics-lock-dir",
                     'PASSWORD_MIN_LENGTH': 6,
-                    'PASSWORD_MIN_ZXCVBN_QUALITY': 0.5,
+                    'PASSWORD_MIN_ENTROPY_BITS': 16,
```

Comment:
Let's separate changing the mechanism from changing the substantive standard -- so in this commit, set the default to be the new-units equivalent of the old standard. By my calculation, that would be 29 bits.

Then in another commit, we'll change the threshold to 16 bits (or so.)